src/frontendproxy/envoy.tmpl.yaml

```diff
@@ -36,6 +36,8 @@ static_resources:
                           route: { cluster: featureflag, prefix_rewrite: "/" }
                         - match: { prefix: "/loadgen" }
                           route: { cluster: loadgen, prefix_rewrite: "/" }
+                        - match: { prefix: "/oltp-http/" }
```

Comment:
I need a trailing splash here.
Without it, envoy returns 301 and redirects to the path without `/oltp-http`